reference/promise-types/packages.markdown

```diff
@@ -526,3 +526,34 @@ Manage packages using Slackware's [slackpkg](https://slackpkg.org).
 **History:**
 
 * Added in CFEngine 3.12.0
+
+### msiexec
+
+Manage MSI packages using MSI installer on Windows.
+
+Due to lack of central software repository on supported versions of Windows,
+neither installation from repository nor checking for upgrades is not supported.
+You need to provide full path to MSI pakage file to install, or installed
```

Comment:
The full path to the MSI package file must be supplied in order to promise the package is installed. In order to promise a package is absent (not installed) the package name must be used.